packages/rn-tester/.maestro/legacy-native-module.yml

```diff
@@ -0,0 +1,46 @@
+appId: ${APP_ID} # iOS: com.meta.RNTester.localDevelopment | Android: com.facebook.react.uiapp
+---
+- launchApp
+- assertVisible:
+    text: "APIs"
+- tapOn:
+    id: "apis-tab"
+- scrollUntilVisible:
+    element:
+        id: "Legacy Native Module"
+    direction: DOWN
+    speed: 40
+- tapOn:
+    id: "Legacy Native Module"
+- tapOn: 
+    text: "voidFunc"
```

Comment:
You don't need to `tapOn` all the buttons. Instead you should be able to click on the "Run all tests" on top and then assert the single results

![Screenshot_1747650231](https://github.com/user-attachments/assets/e5b01ca5-a5e2-40c2-bb07-dc245b532fed)

Reply:
@cortinico the reason why I didn't go that way was because currently there is a bug in the native modules and pressing "Run all tests" would crash in old arch hence the test would not pass. I think that would be indeed the way to go once that bug is fixed (unless we can target this specific test for new arch only)

Reply:
> pressing "Run all tests" would crash in old arch hence the test would not pass

Oh what's the bug 🤔